src/gui/editorwidgets/qgsvaluemapconfigdlg.cpp

```diff
@@ -387,7 +387,9 @@ void QgsValueMapConfigDlg::loadMapFromCSV( const QString &filePath )
   }
 
   QTextStream s( &f );
-  s.setAutoDetectUnicode( true );
```

Comment:
Why was this removed?

Reply:
Thanks for reviewing!
`setAutoDetectUnicode` is not used anywhere else in the code, so I though it was used by mistake 11 years ago https://github.com/qgis/QGIS/commit/ea91b6fe4a21ff741de918a11d0f2a26a291c1e2 in place of `setCodec( "UTF-8" )`.